ACHNBrowserUI/ACHNBrowserUI/views/villagers/VillagerDetailView.swift

```diff
@@ -75,6 +78,18 @@ struct VillagerDetailView: View {
                 .font(.subheadline)
         }.listRowBackground(Rectangle().fill(backgroundColor))
     }
+
+    private var makeCloseButton: some View {
+        if isPresentedInModal {
+            return Button(action: { self.presentation.wrappedValue.dismiss() }) {
+                Image(systemName: "xmark.circle.fill")
+                    .style(appStyle: .barButton)
+                    .foregroundColor(.acText)
+            }.eraseToAnyView()
```

Comment:
I would recommend modifying the close button so it will look like our other close buttons (please test this, I wrote these changes on GitHub website):
```suggestion
            return Button(action: { self.presentation.wrappedValue.dismiss() }, label: {
                Image(systemName: "xmark.circle.fill")
                    .style(appStyle: .barButton)
                    .foregroundColor(.acText)
            })
            .buttonStyle(BorderedBarButtonStyle())
            .accentColor(Color.acText.opacity(0.2))
            .safeHoverEffectBarItem(position: .leading)
            .eraseToAnyView()
```

Reply:
Your code is good, but we need to `import SwiftUIKit` as well. So I will push a new commit